src/context/UnsupportedPropertiesContext/UnsupportedPropertiesContext.example.md

```diff
@@ -0,0 +1,116 @@
+<!--
+ * Released under the BSD 2-Clause License
+ *
+ * Copyright © 2018-present, terrestris GmbH & Co. KG and GeoStyler contributors
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * * Redistributions of source code must retain the above copyright notice,
+ *   this list of conditions and the following disclaimer.
+ *
+ * * Redistributions in binary form must reproduce the above copyright notice,
+ *   this list of conditions and the following disclaimer in the documentation
+ *   and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ *
+-->
+
+The `UnsupportedPropertiesContext` deploys unsuported properties to underlying components.
+The components themeselve will disable field that are marked as not supported and add notes
+for fields that are only partialy supported.
+
+#### Example
+
+Provide some default values.
+
+```jsx
+import * as React from 'react';
+import { UnsupportedPropertiesContext, FillEditor } from 'geostyler';
+import OlStyleParser from 'geostyler-openlayers-parser';
+import SldStyleParser from 'geostyler-sld-parser';
+import { Checkbox, Switch, InputNumber } from 'antd';
+
+class UnsupportedPropertiesContextExample extends React.Component {
```

Comment:
Out of curiosity: Is there a reason you are using class components instead of function components?

Reply:
When writing the Example i thought that only class components are supported as example. :/